demo/App.vue

```diff
@@ -28,6 +28,8 @@ v-app.app
                     v-model="rootLocationDisabled")
                 v-divider.mt-2.mb-1
                 v-checkbox.mt-1.mb-1(hide-details, label="Search Box", v-model="searchEnabled")
+                v-checkbox.mt-1.mb-1(hide-details, label="Dark theme",
```

Comment:
In my testing, `v-model="$vuetify.theme.dark"` worked.

Reply:
Good catch.